stablehlo/tests/verify_reduce_window.mlir

```diff
@@ -742,39 +544,127 @@ func.func @reduce_window_invalid_reducer(%arg0: tensor<4x2xf32>, %init0: tensor<
 
 // -----
 
-func.func @reduce_window_invalid_reducer(%arg0: tensor<4x2xf32>, %init0: tensor<4x2xf32>)
-  -> tensor<2x1xf32> {
-  %0 = "stablehlo.reduce_window"(%arg0, %init0) ({
-         ^bb0(%a0: tensor<4x2xf32>, %b0: tensor<4x2xf32>):
-              %2 = stablehlo.add %a0, %b0 : tensor<4x2xf32>
-              "stablehlo.return"(%2) : (tensor<4x2xf32>) -> ()
+func.func @reduce_window_i2(%arg0: tensor<4x2xf32>, %arg1: tensor<4x2xi32>,
```

Comment:
This seems more like c13 test. The correspondence between result and element type of input (which equal to init_value as per c3) is testing in c13

Reply:
We can have a dedicated test once we have the check for i2.

Reply:
> We can have a dedicated test once we have the check for i2.
See below comment on #704